lnwallet/channel.go

```diff
@@ -3831,6 +3845,11 @@ func (lc *LightningChannel) SignNextCommitment() (lnwire.Sig, []lnwire.Sig,
 	}
 	lc.sigPool.SubmitSignBatch(sigBatch)
 
+	// Ensure that we're passing the correct derivation info to the signer,
+	// and clean up after ourselves.
```

Comment:
Would be great to add some context here why this clean up is necessary? Is the sign descriptor re-used outside of this method?